### PR TITLE
Add existing plugins

### PR DESCRIPTION
**Pull request by yantonov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +63,27 @@ See how to use plugins in plugin documentation.
 #### C/C++
 
 - [dogebuild-c](https://github.com/dogebuild/dogebuild-c) - a C/C++ build plugin
+
+#### Go lang
+
+- [dogebuild-go](https://github.com/dogebuild/dogebuild-go) - a Go lang build plugin
```
Comment (kirillsulim) [inline README.md]: Go plugin is still in development phase, so I would prefer to not include it list of plugins.

Comment (yantonov) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -63,3 +63,27 @@ See how to use plugins in plugin documentation.
...
+
+#### Free Pascal
+
+- [dogebuild-fpc](https://github.com/dogebuild/dogebuild-fpc) - a Free Pascal build plugin
```
Comment (kirillsulim) [inline README.md]: FPC plugin not adapted to current dogebuild state. I should archive it or adapt. However adaptation should be simple.

File: README.md [hunk 3]
```diff
@@ -63,3 +63,27 @@ See how to use plugins in plugin documentation.
...
+
+
+#### Tex plugin
+
+- [dogebuild-tex](https://github.com/dogebuild/dogebuild-tex) - a Tex plugin
+
+
+#### Hello world plugin
+
+- [dogebuild-hello](https://github.com/dogebuild/dogebuild-hello) - Hello world plugin
```
Comment (kirillsulim) [inline README.md]: Hello world plugin is the same as FPC. I think there is no point to keep it as far as dogebuild has documentation and tapas generators. I will archive it.

Comment (yantonov) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (yantonov) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (kirillsulim) [reply README.md]: I have added links to documentation and tapa and archived repository of hello world plugin

**Pull request merged**